src/components/todoItem.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import cn from 'classnames';
+
+interface Props {
+  todo: Todo;
+  loading: number[] | null;
+  onPatch: (id: number, data: Partial<Todo>) => void;
+  onDelete?: (id: number[]) => void;
+}
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  loading,
+  onPatch,
+  onDelete = () => {},
+}) => {
+  return (
+    <div
+      data-cy="Todo"
+      key={todo.id}
+      className={cn('todo', 'item-enter-done', { completed: todo.completed })}
+    >
+      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control*/}
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
+          onChange={() => onPatch(todo.id, { completed: !todo.completed })}
+        />
+      </label>
+      <span data-cy="TodoTitle" className="todo__title">
+        {todo.title}
+      </span>
+      <button
+        type="button"
+        className="todo__remove"
+        data-cy="TodoDelete"
+        onClick={() => onDelete([todo.id])}
```

Comment:
```suggestion
        onClick={() => onDelete(todo.id)}
```
are you throwing in a few ids somewhere to use array here?

Reply:
![image](https://github.com/user-attachments/assets/a9e05ac5-3e9f-4752-8f85-6d7a13463f8b)